src/main/scala/mesosphere/marathon/state/LinuxInfo.scala

```diff
@@ -0,0 +1,40 @@
+package mesosphere.marathon
+package state
+
+import com.wix.accord._
+
+/**
+  * Defines the seccomp expectations for the instance this is associated with.   This will govern the secure computing mode.
+  * Unlike Mesos, unconfined is not optional.  If seccomp is provided, then unconfined is either true or false.  The optionality is with seccomp itself.
```

Comment:
I get this, but why is then `unconfined` optional in our `linux.raml`?

Reply:
good point!  if we are defining `seccomp`.   It should be required.   Changed based on previous comment!